model-optimizer/extensions/middle/AddMeanScaleValues_test.py

```diff
@@ -44,7 +46,19 @@
 }
 
 
+def set_graph_attrs(graph, graph_ref):
```

Comment:
The function should have just one parameter - the graph to set attrs. And it will be called twice in each test: for the `graph` and `graph_ref`.